package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.7.0",
   "description": "Collection of Resin.io JavaScript errors",
   "main": "build/errors.js",
+  "types": "build/errors.d.ts",
```

Comment:
At least our styleguide mentions the .d.ts file. Will update to include the TS sources and point at them.